packages/mongodb-chatbot-ui/src/DevCenterChatbot.tsx

```diff
@@ -1,6 +1,6 @@
 import { useDarkMode } from "@leafygreen-ui/leafygreen-provider";
 import { DarkModeProps } from "./DarkMode";
-import { FloatingActionButtonTrigger, FloatingActionButtonTriggerProps } from "./FloatingActionButtonTrigger";
+import { FloatingActionButtonTrigger, type FloatingActionButtonTriggerProps } from "./FloatingActionButtonTrigger";
```

Comment:
?

Reply:
consider removing the `type` so this file doesn't change

Reply:
What's your question here?